src/IRCSock.cpp

```diff
@@ -1323,7 +1323,7 @@ void CIRCSock::Connected() {
     PutIRC("CAP LS 302");
 
     if (!sPass.empty()) {
-        PutIRC("PASS " + sPass);
+        PutIRC("PASS :" + sPass);
```

Comment:
This will break broken servers which don't expect PASS to start with `:` and just take the second word as is

Need to add it or not depending on what characters the password has; and easy way to do that is to use CMessage

Reply:
I think the CString overload of PutIRC will already use CMessage?

Reply:
When CMessage parses a string, it remember whether the trailing arg had an extra colon, and if it had, it puts the extra colon there when stringifying.

Reply:
But when you do SetParam(), that flag is not set